src/app/components/toolbar/toolbar-guidelines.component.html

```diff
@@ -240,8 +239,6 @@
     </stache-column>
   </stache-row>
 
-  <div class="screenshot screenshot-info"><img src="~/assets/img/guidelines/toolbar/more-button-dropdown.png" style="width: 100%;" /></div>
-
   <div class="sky-subsection-heading">Find in this list</div>
   <p>
     When a toolbar includes a search component, it should follow the actions and more actions button. The search field is aligned with the action buttons, so if a user has a really large monitor resolution, the field appears on the left side of the screen.
```

Comment:
One final question about existing text: In the "Items related to views" section after this, the first sentence is "The view section of the toolbar is docked on the right side of the screen." Is "screen" accurate? Or should it be "toolbar"? Seems like the latter to me, but I hadn't noticed that before.

Reply:
Should be "toolbar". And "docked" implies some functionality that isn't actually there. Could say something like "The section for items that affect the view of a list is on the right side of the toolbar"